nifi-registry/nifi-registry-core/nifi-registry-jetty/src/main/java/org/apache/nifi/registry/jetty/JettyServer.java

```diff
@@ -147,9 +151,39 @@ private File getDocsDir(final String docsDirectory) {
     }
 
     private void configureConnectors() {
-        final ServerConnectorFactory serverConnectorFactory = new ApplicationServerConnectorFactory(server, properties);
-        final ServerConnector serverConnector = serverConnectorFactory.getServerConnector();
-        server.addConnector(serverConnector);
+        try {
+            final ServerConnectorFactory serverConnectorFactory = new ApplicationServerConnectorFactory(server, properties);
+            final Map<String, String> interfaces = properties.isHTTPSConfigured() ? properties.getHttpsNetworkInterfaces() : Collections.emptyMap();
+            final Set<String> interfaceNames = interfaces.values().stream().filter(StringUtils::isNotBlank).collect(Collectors.toSet());
+            if (properties.isHTTPSConfigured() && !interfaceNames.isEmpty()) {
```

Comment:
Recommend simplifying and reversing the logic. The `properties.isHTTPConfigured()` check is called above, so it does not need to be checked again.
```suggestion
            if (interfaceNames.isEmpty()) {
                // serverConnectorFactory.getServerConnector()
            } else {
                // stream through interface names
            }
```

Reply:
Making the change